aws/table_aws_inspector_assessment_template.go

```diff
@@ -220,6 +225,50 @@ func getAwsInspectorAssessmentTemplateTags(ctx context.Context, d *plugin.QueryD
 	return op, nil
 }
 
+// API call for fetching event subscriptions
+func listAwsInspectorAssessmentEventSubscriptions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("ListAwsInspectorAssessmentTemplateEventSubscriptions")
+
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	assessmentTemplateArn := *h.Item.(*inspector.AssessmentTemplate).Arn
+
+	// Create Session
+	svc, err := InspectorService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &inspector.ListEventSubscriptionsInput{
+		ResourceArn: &assessmentTemplateArn,
+	}
+	// Create a Struct name EventSubscriptions as the framework will use the field name to create
+	// a key from which it extracts the contents of the field
+	type AssociatedEventSubscriptions struct {
```

Comment:
@davidhammturner For your earlier question, we don't need a `struct` here; instead you can store the `[]*inspector.Subscription` data in a variable and return that, e.g.,
```
var eventSubscriptions []*inspector.Subscription
```

I've also included a suggestion in the column definition that uses `transform.FromValue`, which tells the column to just use the returned data (more information on these transform functions can be found at https://steampipe-io-git-staging-turbot.vercel.app/docs/develop/writing-plugins#transform-functions).